Checkout/Checkout/Source/Model/Country.swift

```diff
@@ -270,4 +276,12 @@ public struct Country: Equatable {
       Country(iso3166Alpha2: "ZW", dialingCode: "263")
     ]
   }
+
+  public static func from(iso3166Alpha2: String) -> Country? {
+    return allAvailable.first { $0.iso3166Alpha2 == iso3166Alpha2 }
+  }
+
+  public static func from(dialingCode: String) -> [Country] {
+    return allAvailable.filter { $0.dialingCode == dialingCode }
+  }
```

Comment:
This method appears to not be used anywhere. 

If we keep, at reading site, would we expect
`Country.from(dialingCode: code)` to return a list of countries? Semantics aside, is it a fair assumption for this to return an optional country as dialingCode should be unique to each country, in the way we expect `iso3166Alpha2` to be unique?